src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
```

Comment:
Note too that if we get to the end of the migration, we'll mark the queue as "migrated" and never try again.